src/pages/index.js

```diff
@@ -18,7 +18,12 @@ export default function SiteIndex({ data }) {
 
   const contests = data.contests.edges;
   const filteredContests = contestsByState({ contests });
-
+  // TODO change this to fetch from notion
```

Comment:
Doesn't it? And things seem to go weird with contest/node structure.

Reply:
Yes right I didn't do it here, I'll do it now

Reply:
Done :-)